language-service/language_service/dto/__init__.py

```diff
@@ -0,0 +1,29 @@
+class Definition:
+    def __init__(self, subdefinitions=[], tag="", examples=[], optional_fields=[]):
+        self.subdefinitions = subdefinitions
+        self.tag = tag
+        self.examples = examples
+        self.optional_fields = optional_fields
+
+    def set_subdefinitions(self, subdefinitions):
+        self.subdefinitions = subdefinitions
+
+    def set_optional_fields(self, optional_fields):
+        self.optional_fields = optional_fields
+
+    def to_json(self):
+        return {
+            "subdefinitions": self.subdefinitions,
+            "tag": self.tag,
+            "examples": self.examples,
+            "optional_fields": self.optional_fields,
+        }
+
+
+class Word(dict):
+    def __init__(self, token="", tag="", lemma=""):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)